lib/avo/engine.rb

```diff
@@ -72,6 +72,11 @@ class Engine < ::Rails::Engine
       require_relative "../generators/model_generator"
     end
 
+    initializer "avo.locales" do |app|
+      I18n.load_path += Dir[Avo::Engine.root.join("lib", "generators", "avo", "templates", "locales", "*.{rb,yml}")]
+      I18n.load_path += Dir[Rails.root.join("config", "locales", "*.{rb,yml}")]
```

Comment:
Any reason why this is added? Engine should not care about `Rails.root` files at all. This breaks rubygems.org [tests](https://github.com/rubygems/rubygems.org/actions/runs/5136907592/jobs/9244269425?pr=3827).

Reply:
Hey @simi. Thanks for writing.

When installing Avo a bunch of locales were being copied into the parent app.

 - we try to keep the number of files we add as little as possible
 - those locales are not auto-updatable. They only update when the developer runs `rails generate avo:locales` (which rarely happens).

I can't quite understand what broke from those tests, but I'd love to help fix it. 
Do you have any ideas why the tests fail? Do we need to do some kind of a `defined?(Rails)` check?

Reply:
These lines seem also to be breaking `clearance` gem's I18n for me as well -- I'm not able to override clearanaces language with the latest avo versions.

Reply:
I'm sorry, but I do have not a much time to explore these days. But can you explain why `I18n.load_path += Dir[Rails.root.join("config", "locales", "*.{rb,yml}")]` is needed? Why do you need to add apps locales to `load_path`? I understand the line above, load Avo locales, but I don't follow this one.

Reply:
@simi we added that so you can add your own overrides in you're `config/locales` directory.
Otherwise, if we don't add it you have to add it yourself somewhere in `application.rb`.

Can you please try `'avo', '2.34.4.pre.1'` and see if that removes the CI crash.
I removed the `I18n.load_path += Dir[Rails.root.join("config", "locales", "*.{rb,yml}")]` statement.

Reply:
@simi released `2.34.4` now with that fix.